cs-amazon/src/main/java/io/cloudslang/content/amazon/factory/helpers/InstanceUtils.java

```diff
@@ -346,12 +346,12 @@ private void setFilterValues(Map<String, String> queryParamsMap, String filterNa
         }
     }
 
-    @NotNull
+    @Contract(pure = true)
```

Comment:
I don't really like to add this kind of third party annotations to the code - I can leave with it